[harfbuzz] Missing INTERFACE_INCLUDE_DIRECTORIES for harfbuzz::harfbuzz target

**Describe the bug**

Compiler complains that it cannot find `<hb.h>` despite installing harfbuzz with vcpkg, and linking to the target in CMake.

**Environment**
- OS: Windows 7 64bit
- Compiler: MSVC 2017 x64

**To Reproduce**

1. `vcpkg integrate install`
2. `vcpkg install freetype:x64-windows harfbuzz:x64-windows`

CMakeLists.txt:
```
cmake_minimum_required(VERSION 3.1.0)
project(testharfbuzz)
set(CMAKE_CXX_STANDARD 11)

find_package(freetype CONFIG REQUIRED)
get_target_property(FREETYPE_INCLUDE_DIRS freetype INTERFACE_INCLUDE_DIRECTORIES)
message(STATUS "FreeType include dirs: ${FREETYPE_INCLUDE_DIRS}")

find_package(harfbuzz CONFIG REQUIRED)
get_target_property(HARFBUZZ_INCLUDE_DIRS harfbuzz::harfbuzz INTERFACE_INCLUDE_DIRECTORIES)
message(STATUS "HarfBuzz include dirs: ${HARFBUZZ_INCLUDE_DIRS}")

add_executable(main main.cpp)
target_link_libraries(main PRIVATE freetype harfbuzz::harfbuzz)
```

main.cpp:
```
#include <ft2build.h>
#include FT_FREETYPE_H
#include FT_OUTLINE_H

#include <hb.h>
#include <hb-ft.h>

int main() {
	return 0;
}
```

3. `cmake .. -G "Visual Studio 15 2017" -A x64 -DCMAKE_TOOLCHAIN_FILE="%UserProfile%\vcpkg\scripts\buildsystems\vcpkg.cmake"`

Output:
```
-- Selecting Windows SDK version 10.0.17134.0 to target Windows 6.1.7601.
-- The C compiler identification is MSVC 19.15.26729.0
-- The CXX compiler identification is MSVC 19.15.26729.0
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.15.26726/bin/Hostx86/x64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.15.26726/bin/Hostx86/x64/cl.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.15.26726/bin/Hostx86/x64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.15.26726/bin/Hostx86/x64/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- FreeType include dirs: C:/Users/Boris/vcpkg/installed/x64-windows/include
-- HarfBuzz include dirs: HARFBUZZ_INCLUDE_DIRS-NOTFOUND
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Boris/Documents/testharfbuzz/build
````

Expected output:
```
-- HarfBuzz include dirs: C:/Users/Boris/vcpkg/installed/x64-windows/include/harfbuzz
````

4.  `cmake --build . --config Release`


Output:
```
c:\users\boris\documents\testharfbuzz\main.cpp(6): fatal error C1083: Cannot open include file: 'hb.h': No such file or directory [C:\Users\Boris\Documents\testharfbuzz\build\main.vcxproj]
````

Expected output: No errors

**Additional context**

I suspect there's an INTERFACE_INCLUDE_DIRECTORIES something missing in the vcpkg\installed\x64-windows\share\harfbuzz\harfbuzzConfig.cmake

Indeed, we can see the following:

```
# Create imported target harfbuzz::harfbuzz
add_library(harfbuzz::harfbuzz SHARED IMPORTED)
```

But there seems to be no INTERFACE_INCLUDE_DIRECTORIES added. By comparison, we can see the following in vcpkg\installed\x64-windows\share\freetype\freetype-config.cmake:

```
# Create imported target freetype
add_library(freetype SHARED IMPORTED)

set_target_properties(freetype PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
)
```

--- a/ports/harfbuzz/CONTROL
+++ b/ports/harfbuzz/CONTROL
@@ -1,5 +1,6 @@
 Source: harfbuzz
 Version: 2.6.6
+Port-Version: 1
 Description: HarfBuzz OpenType text shaping engine
 Homepage: https://github.com/behdad/harfbuzz
 Build-Depends: freetype[core], ragel, gettext (osx)

--- a/ports/harfbuzz/fix_include.patch
+++ b/ports/harfbuzz/fix_include.patch
@@ -1,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c4dc524..bfd15ec 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -431,6 +431,7 @@ endif ()
+ 
+ ## Define harfbuzz library
+ add_library(harfbuzz ${project_sources} ${project_extra_sources} ${project_headers})
++target_include_directories(harfbuzz PUBLIC $<INSTALL_INTERFACE:include/harfbuzz>)
+ target_link_libraries(harfbuzz ${THIRD_PARTY_LIBS})
+ 
+ 

--- a/ports/harfbuzz/portfile.cmake
+++ b/ports/harfbuzz/portfile.cmake
@@ -12,6 +12,7 @@ vcpkg_from_github(
         find-package-freetype-2.patch
         # This patch is required for propagating the full list of dependencies from glib
         glib-cmake.patch
+        fix_include.patch
 )
 
 file(READ ${SOURCE_PATH}/CMakeLists.txt _contents)

